.github/workflows/conflicts.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Check for merge conflicts'
+on:
+  push:
+    branches:
+      - master
+jobs:
+  triage:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: mschilde/auto-label-merge-conflicts@master
```

Comment:
This isn't a repository we control, and so we shouldn't be on their master.

Change this to a direct commit.

Reply:
Ideally we'd just be on their tag, but you can update those remotely so meh.